### PR TITLE
KDE Frameworks 5.114

### PR DESCRIPTION
**Pull request by ReillyBrogan**
**Summary**
- Updated KDE Frameworks to v5.114
- Fixed all licenses that were misspelled
- Ordered all build and rundeps correctly
- Went through all packages with .qml files and made sure we were building pre-cached versions (only kcmutils affected)

Release Notes:
- [5.114](https://kde.org/announcements/frameworks/5/5.114.0/)
- [5.113](https://kde.org/announcements/frameworks/5/5.113.0/)

**Test Plan**
- Tested a Plasma VM and everything looked fine

### CODE REVIEW

File: packages/k/ktextwidgets/package.yml
Comment (Justinzobel) [inline packages/k/ktextwidgets/package.yml:1]: It says the LGPL part here was changed, but I don't see any change?

Comment (ReillyBrogan) [reply packages/k/ktextwidgets/package.yml:1]: The previous license was misspelled. `LPGL` is not a valid license identifier, it should have been `LGPL`.

Comment (Justinzobel) [reply packages/k/ktextwidgets/package.yml:1]: Ah derp, my eyes doth deceive me.

**Review by Justinzobel — APPROVED** [review]
LGTM

**Review by Justinzobel — APPROVED** [review]

**Pull request merged**